dnx/src/Microsoft.DotNet.xunit.performance.runner.dnx/CSVMetricLogger.cs

```diff
@@ -18,6 +20,29 @@ internal class CSVMetricLogger : IPerformanceMetricLogger
         public CSVMetricLogger(XunitPerformanceProject project)
         {
             _csvPath = Path.GetFullPath(Path.Combine(project.OutputDir, project.RunId + ".csv"));
+
+            var diagnosticMessageSink = new ConsoleReporter();
+
+            foreach (var assembly in project.Assemblies)
+            {
+                Console.WriteLine($"Discovering tests for {assembly.AssemblyFilename}.");
+
+                // Note: We do not use shadowCopy because that creates a new AppDomain which can cause
```

Comment:
Is this comment accurate on CoreCLR? I mean, AFAIK, CoreCLR doesn't support AppDomains anyway.